piqueserver/config/config.toml

```diff
@@ -196,6 +196,10 @@ color = [ 0, 255, 0]
 #guard = ["guardpass"]
 #trusted = ["trustedpass"]
 
+# Include an empty password for the admin role to automatically log in everyone
+# as admin. Could be useful for testing.
+#admin = [""]
```

Comment:
I don't like this, as it's implicit and could happen accidentally, which is why we removed this behavior initially. I'd prefer a designated everyone_is_admin option. I thought we already had that, but seems not.

Reply:
i'm not sure we ever did remove the behaviour. it was still there, just one /reloadconfig away...